### PR TITLE
more cryptocurrency apis

### PR DESCRIPTION
**Pull request by bittricky**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [x] Your additions are ordered alphabetically
- [x] Your submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column should be padded with one space on either side
- [x] You have searched the repository for any relevant issues or pull requests
- [ ] Any category you are creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -179,15 +179,19 @@ API | Description | Auth | HTTPS | CORS |
 | [CoinLayer](https://coinlayer.com) | Real-time Crypto Currency Exchange Rates | `apiKey` | Yes | Unknown |
 | [CoinMarketCap](https://coinmarketcap.com/api/) | Cryptocurrencies Prices | No | Yes | Unknown |
 | [Coinpaprika](https://api.coinpaprika.com) | Cryptocurrencies prices, volume and more | No | Yes | Yes |
+| [CoinRanking](https://docs.coinranking.com/) | Live Cryptocurrency data | No | Yes | Unknown |
 | [CryptoCompare](https://www.cryptocompare.com/api#) | Cryptocurrencies Comparison | No | Yes | Unknown |
 | [Cryptonator](https://www.cryptonator.com/api/) | Cryptocurrencies Exchange Rates | No | Yes | Unknown |
 | [Gemini](https://docs.gemini.com/rest-api/) | Cryptocurrencies Exchange | No | Yes | Unknown |
+| [ICObench](https://icobench.com/developers) | Various information on listing, ratings, stats, and more | `apiKey` | Yes | Unknown |
```
Comment (davemachado) [inline README.md]: Is this a free service? I could only find record of "Premium" access.

File: README.md [hunk 2]
```diff
@@ -179,15 +179,19 @@ API | Description | Auth | HTTPS | CORS |
...
 | [Livecoin](https://www.livecoin.net/api) | Cryptocurrency Exchange | No | Yes | Unknown |
 | [MercadoBitcoin](https://www.mercadobitcoin.net/api-doc/) | Brazilian Cryptocurrency Information | No | Yes | Unknown |
 | [Nexchange](https://nexchange2.docs.apiary.io/) | Automated cryptocurrency exchange service | No | No | Yes |
 | [NiceHash](https://www.nicehash.com/doc-api) | Largest Crypto Mining Marketplace | `apiKey` | Yes | Unknown |
 | [Poloniex](https://poloniex.com/support/api/) | US based digital asset exchange | `apiKey` | Yes | Unknown |
+| [Voltaire](https://docs.voltaire.cash/) | Order Management, Account Management, Market Data, and more | `apiKey` | Yes | Unknown |
```
Comment (davemachado) [inline README.md]: Also could find documentation to free tier of API.

File: README.md (hunk 1 above)
Comment (bittricky) [reply README.md]: It should be. Under the Limitation of Liability it claims this:

"The API is being provided free of charge. Accordingly, you agree that ICObench shall have no liability arising from, relating to or based on your use of the API."

File: README.md (hunk 2 above)
Comment (bittricky) [reply README.md]: Looking at it at the terms & conditions for the service there is no free tier of the API. I'll remove the link from the section.

**Comment by davemachado** [comment]
Thanks!

**Pull request merged**